crates/nu-cli/src/repl.rs

```diff
@@ -263,6 +263,11 @@ pub fn evaluate_repl(
         }
 
         let input = line_editor.read_line(prompt);
+
+        // Just before running a command/program, send the escape code (see
+        // https://sw.kovidgoyal.net/kitty/shell-integration/#notes-for-shell-developers)
+        print!("\x1b]133;C\x1b\\");
```

Comment:
I'll bet that this is the wrong place for printing this here. Here would some guidance helpful.